templates/supplier_app/Supplier/taxpayer-creation.html

```diff
@@ -75,22 +72,22 @@ <h3 class="fs-title mt-3">- {% trans "Address Info" %} -</h3>
             </div>
         </div>
 
-        <h3 class="fs-title mt-5">- {% trans "Contact Info" %} -</h3>
+        <h3 class="fs-title mt-3 mb-4">- {% trans "Contact Info" %} -</h3>
         <div class="text-left">{{ contact_form.contact_person.label_tag }}</div>
         <div class="div-input mandatory">{{ contact_form.contact_person }}</div>
         <div class="text-left">{{ contact_form.phone_number.label_tag }}</div>
         <div class="div-input mandatory">{{ contact_form.phone_number }}</div>
         <div class="row">
             <div class="col">
                 <div class="text-left">{{ contact_form.email.label_tag }}</div>
-                <div class="div-input mandatory">{{ contact_form.email }}</div>
+                <div class="div-input mandatory special-rules">{{ contact_form.email }}</div>
             </div>
             <div class="col">
                 <div class="text-left">{{ contact_form.website.label_tag }}</div>
                 <div class="div-input optional">{{ contact_form.website }}</div>
             </div>
         </div>
-        <input type="button" name="next"  class="next action-button" value="{% trans 'Next' %}" />
+        <input type="button" name="next"  class="next action-button" id="coso" value="{% trans 'Next' %}" />
```

Comment:
NIT: Try always to choose more representative names

Reply:
That was a placeholder for something i was trying and forgot to delete sorry!